src/controller/userController.js

```diff
@@ -0,0 +1,6 @@
+exports.getUser = async (req, res, next) => {};
+exports.addUser = async (req, res, next) => {};
+exports.updateUser = async (req, res, next) => {};
+exports.deleteUser = async (req, res, next) => {};
+exports.getRecords = async (req, res, next) => {};
+exports.deleteRecords = async (req, res, next) => {};
```

Comment:
Could we export this as an object at the end? I don't like that way, but that is very opinionated, it is just a matter of taste, I really prefer doing: `module.exports = {getUser, addUser, ...}` Is that doable or you hate it?

Reply:
We would have to remove the exports from in front of each function.

Reply:
I have no feeling about it, changed it in your preferred way